interface_frontend.py

```diff
@@ -35,11 +34,14 @@ def get_sprite(img_path, x=0, y=0):
 select_sprite = get_sprite('img/interface/png/card_cv.png')
 # Selection cursor
 cursor_sprite = get_sprite('img/interface/png/card_sl.png')
+# Other robot card
+players_background = get_sprite('img/interface/png/player.png')
 # Loading of robots images
 loaded_robots_images = {}
 for image_path in Path('./img/robots/png').iterdir():
     loaded_robots_images[image_path.stem] = pyglet.image.load(image_path)
-    
+
+player_sprite = get_sprite('img/robots/png/bender.png')
```

Comment:
Dvakrát po sobě jsou zde načtené obrázky stejného robota. Vím, proč tak je, protože jsme se o tom bavily. Jelikož to není úplně jasné na první pohled, uvítala bych zde komentář s vysvětlením, že se jedná o náhodný obrázek pro vytvoření sprite, který se dále v kódu nahradí skutečným.